server/boards/routes.ts

```diff
@@ -130,6 +131,60 @@ router.get("/:board_id", ensureBoardAccess, async (req, res) => {
     );
 });
 
+/**
+ * @openapi
+ * /boards:
+ *   post:
+ *     summary: Create board.
+ *     operationId: createBoard
+ *     tags:
+ *       - /boards/
+ *     requestBody:
+ *       description: request body
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: "#/components/schemas/CreateBoardMetadata"
+ *     responses:
+ *       401:
+ *         $ref: "#/components/responses/ensureLoggedIn401"
+ *       403:
+ *         $ref: "#/components/responses/ensureBoardPermission403"
+ *       500:
+ *         description: Internal Server Error
+ *         $ref: "#/components/responses/default500"
+ *       200:
+ *         description: Metadata of the created board.
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: "#/components/schemas/LoggedInBoardMetadata"
+ *             examples:
+ *               existing:
+ *                 $ref: "#/components/examples/GoreMetadataUpdateResponse"
+ */
+router.post(
+  "/",
+  ensureLoggedIn,
+  // TODO ensureBoardPermission()
+  async (req, res) => {
+    const { descriptions, accentColor,  avatar_url, tagline} = req.body;
+
+    const success = await createBoard({
```

Comment:
Since your query is doing `RETURNING *`, this isn't really `success` as much as it is `newBoardData`, but also since it's simply returning everything in the query it's going to 
a) leak internal details, like board id
b) not match what the OpenAPI definition expects, most likely

It's not that huge of a deal at this stage and in this case, but I thought I'd mention it.

You'd likely want to do:

```ts
const newBoardData = await createBoard(...);



   // Explicitly set all the values from newBoard data, so that extra details won't leak
   // I don't remember what these are so i made up a name
    res.status(200).json({
        board_name: newBoardData.board_name,
        ....
    }};
```